tests/storage/restore.go

```diff
@@ -1646,11 +1654,16 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					vm, vmi = createAndStartVM(vm)
 
 					checkCloneAnnotations(vm, true)
+					if deleteSourcePVC {
+						libstorage.DeleteDataVolume(&sourceDV)
+					}
 					doRestore("", console.LoginToCirros, offlineSnaphot, getTargetVMName(restoreToNewVM, newVmName))
 					checkCloneAnnotations(getTargetVM(restoreToNewVM), false)
 				},
-					Entry("to the same VM", false),
-					Entry("to a new VM", true),
+					Entry("to the same VM", false, false),
```

Comment:
I'd add the full description here, you are simply describing only one of the 2 booleans. We shouldn't have 2 entries with the same description

Reply:
right I did it on the first 2 tests and forgot to update this 2 too :) thanks